lib/index.js

```diff
@@ -42,5 +57,10 @@ module.exports = new BinWrapper()
   .src(`${baseUrl}hugo_${hugoVersion}_macOS-64bit.tar.gz`, 'darwin', 'x64')
   .src(`${baseUrl}hugo_${hugoVersion}_Windows-32bit.zip`, 'win32', 'x86')
   .src(`${baseUrl}hugo_${hugoVersion}_Windows-64bit.zip`, 'win32', 'x64')
-  .dest(path.join(__dirname, '../vendor'))
+  .dest(dest)
   .use(binName);
+module.exports = projectRoot => {
+  const config = pkgConf.sync('hugo-bin', { cwd: projectRoot });
+  const extended = (process.env.HUGO_BIN_BUILD_TAGS || process.env.npm_config_hugo_bin_build_tags || config.buildTags || '') === 'extended';
```

Comment:
@satoshun00: I wonder, is the `|| ''` needed? Because later in the `return` statement you check for this anyway.

Reply:
oops  `|| ''` is not needed!